paper.md

```diff
@@ -58,202 +58,195 @@ bibliography: paper.bib
 
 # Description and eligibility
 
-We present an open source course module designed for teaching statistical
-and programming skills in a participatory live-coding format targeting upper year
-undergraduate biology students. Our module has sixteen primarily self-contained
-lectures with topics ranging from data cleaning, manipulation, and visualization,
-to the statistical analysis of biological data and reproducibility through GitHub.
-Our module additionally includes eight assignments that were distributed to
-students throughout the term to assess learning and understanding of the course
-material. All materials were developed and taught using R Markdown [@rmarkdown], although no
-prior knowledge of the R programming language [@R] is assumed prior to taking the
-course. Course content is licensed under the CC-BY 4.0 license while code is
-licensed under MIT. The open-source and stand-alone nature of our computational
-and statistical learning module makes this material ideally suited to adoption
-by other instructors and an excellent contribution to JOSE.
+<!-- this will not be part of the final paper -->
+
+We present an open source learning module suitable for a semester long course
+and designed to leverage participatory live-coding techniques to teach both
+statistical and programming skills to primarily upper-year undergraduate biology
+students. Our learning module has three primarily self-contained submodules spanning
+sixteen lessons: 1) Programming in R, basic data wrangling, and visualizations; 2)
+Exploratory data analysis, statistics, and modelling; and 3) Collaborative and reproducible
+science. Our learning module includes eight assignments, distributed throughout
+the term to assess students' learning and understanding of the material. All
+materials were developed and taught using R Markdown, although no prior
+knowledge of the R programming language is assumed or expected. Our material is
+licensed under CC-BY 4.0 while the code is under a MIT License. The open source
+and modular nature of our computational and statistical learning module, as well
+as responding to and emphasizing on the need for greater programming training
+for researchers, makes our material an ideal resource for other instructors and
+we believe it would be an excellent contribution to JOSE.
 
-# Statement of Need
+# Main Body
```

Comment:
Do we want 'Main Body' as a heading? I don't think it's required, and none of the other papers in JOSE that I saw had it.

Reply:
This is simply to split up the document. Once I move the paper file into the `rcourse` repo, I'll keep only the important sections and remove the Main Body heading.